### DISCUSSION TITLE
Unexpected undeploy of channel?

### DISCUSSION THREAD

**Discussion by RickDavey** [nextgenhealthcare/connect#4686] [Q&A] [answered]
(Is this the correct place and method to ask a question?  Thanks.)

We have corrected this issue, but wondering if this is a normal response so we can watch for it in the future.
We had a channel where the After Processing Action was set to Move.
The Move-to Directory was not mapped in Mirth, so each received message would fail this action.
(Not a busy channel.)
Looking back as far as the Mirth logs go, we see this error occurring.
However, yesterday, there were 11 messages that failed in a short time span.
Would this cause the channel to undeploy itself?
Thanks

**Comment by tonygermano** [comment, marked as answer]
I can't think of any scenarios that would cause a channel to undeploy itself. There are cases where a channel fails to deploy, but once a channel has started, it typically just logs errors when there are problems.

You can click on the Events menu on the left side of the Administrator client and search events with "undeploy" in the name. That will at least show you if a user had undeployed the channel through the client API. Maybe someone didn't realize they had done it.

**Reply by RickDavey** [reply]
Thanks Tony.
It so happens the Mirth Developer meeting was that same day.  I asked on there as well.
They also mentioned the Events log.  (I found the culprit there!)
As well, they mentioned a Java command that would undeploy a channel from another channel.  Not in our case, but good to know.
Thanks again!
